client/stylesheets/_toCat.scss

```diff
@@ -3705,6 +3769,16 @@ On click the images will be displayed at normal size to complete the effect
     text-align: center !important;
 }
 
+.profileUserMore li a i {
+	font-size: 4rem;
+	color: #00bfff;
+}
+
+.profileUserMore li a i:hover {
+	color: #04B2EC !important;
```

Comment:
Color literals like `#04B2EC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#04B2EC` should be written as `#04b2ec`<br>!important should not be used<br>Line should be indented with spaces, not tabs